configuration/api.go

```diff
@@ -13,30 +13,16 @@ import (
 
 	"github.com/mitchellh/mapstructure"
 	"github.com/pkg/errors"
-
-	"github.com/insolar/observer/configuration"
 )
 
-type Configuration struct {
-	Listen               string
-	DB                   configuration.DB
-	FeeAmount            *big.Int
-	PriceOrigin          string
-	Price                string
-	CMCMarketStatsParams CMCMarketStatsParamsEnabled
-	Log                  Log
-}
-
-func (c Configuration) GetConfig() interface{} {
-	return &c
-}
-
-type Log struct {
-	Level        string
-	Format       string
-	OutputType   string
-	OutputParams string
-	Buffer       int
+type ApiConfig interface {
+	GetListen() string
```

Comment:
2.11 Getter name should not contain get prefix.

from here https://insolar.atlassian.net/wiki/spaces/DEV/pages/738131969/Go+code+style

Reply:
But in the case without get prefix we have got errors:
`configuration/api.go:84:6: type Api has both field and method named Listen
configuration/api.go:88:6: type Api has both field and method named DB
configuration/api.go:92:6: type Api has both field and method named Log
`

Reply:
And we cannot make fields in configurations unexported (private) because of the insconfig library.